File: frontend/src/todo-list.js
import {html, PolymerElement} from '@polymer/polymer/polymer-element.js';
import '@vaadin/vaadin-ordered-layout/src/vaadin-vertical-layout.js';
import '@vaadin/vaadin-list-box/src/vaadin-list-box.js';
import '@vaadin/vaadin-item/src/vaadin-item.js';
import '@vaadin/vaadin-ordered-layout/src/vaadin-horizontal-layout.js';

class TodoList extends PolymerElement {

    static get template() {
        return html`
<style include="shared-styles">
                :host {
                    display: block;
                    height: 100%;
                }
            </style>
<vaadin-vertical-layout style="width: 100%; height: 100%;" id="vaadinVerticalLayout" content="Datum">
 <label id="label">Aufgabenliste</label>
 <vaadin-horizontal-layout id="vaadinHorizontalLayout" style="align-self: flex-start;">
  <label id="label1"></label>
  <vaadin-list-box>
   <b>Einmalig</b>
   <vaadin-item>
    Kuchen kaufen
   </vaadin-item>
   <vaadin-item style="flex-grow: 0;">
    Paket beim Post abholen
   </vaadin-item>
   <hr>
  </vaadin-list-box>
  <label style="flex-grow: 0; flex-shrink: 0;"></label>
  <vaadin-list-box id="vaadinListBox">
   <b>Datum</b>
   <vaadin-item>
    05.11.2019
   </vaadin-item>
   <vaadin-item>
    07.11.2010
   </vaadin-item>
   <hr>
   <vaadin-item></vaadin-item>
  </vaadin-list-box>
 </vaadin-horizontal-layout>
 <label id="label2"></label>
 <vaadin-list-box id="vaadinListBox1">
  <b>Woechentlich</b>
  <vaadin-item id="vaadinItem2">
   Gemeinsam Abendessen
   <vaadin-checkbox id="vaadinCheckbox"></vaadin-checkbox>
  </vaadin-item>
  <vaadin-item style="flex-grow: 0;" id="vaadinItem3">
   Spazieren gehen
  </vaadin-item>
  <hr>
 </vaadin-list-box>
 <vaadin-list-box id="vaadinListBox2">
  <b>Monatlich</b>
  <vaadin-item id="vaadinItem">
   Wohnung Pieterlen
   <vaadin-checkbox id="vaadinCheckbox3"></vaadin-checkbox>
  </vaadin-item>
  <vaadin-item style="flex-grow: 0;" id="vaadinItem1">
   Coiffeur Termin
   <vaadin-checkbox id="vaadinCheckbox2"></vaadin-checkbox>
  </vaadin-item>
  <hr id="hr">
 </vaadin-list-box>
 <vaadin-list-box id="vaadinListBox3">
  <b>Jaerlich</b>
  <vaadin-item id="vaadinItem4">
   Steuererklaerung
   <vaadin-checkbox id="vaadinCheckbox1"></vaadin-checkbox>
  </vaadin-item>
  <hr id="hr1">
 </vaadin-list-box>
</vaadin-vertical-layout>
`;
    }

    static get is() {
        return 'todo-list';
    }

    static get properties() {
        return {
            // Declare your properties here.
        };
    }
}

customElements.define(TodoList.is, TodoList);

File: src/main/java/ch/bfh/btx8081/w2019/green/alzman/view/TodoList.java
package ch.bfh.btx8081.w2019.green.alzman.view;

import com.vaadin.flow.templatemodel.TemplateModel;
import com.vaadin.flow.component.Tag;
import com.vaadin.flow.component.dependency.JsModule;
import com.vaadin.flow.component.polymertemplate.PolymerTemplate;
import com.vaadin.flow.component.orderedlayout.VerticalLayout;
import com.vaadin.flow.component.polymertemplate.Id;
import com.vaadin.flow.component.html.Label;
import com.vaadin.flow.component.orderedlayout.HorizontalLayout;
import com.vaadin.flow.dom.Element;
import com.vaadin.flow.component.html.Hr;
import com.vaadin.flow.component.checkbox.Checkbox;

/**
 * A Designer generated component for the todo-list template.
 *
 * Designer will add and remove fields with @Id mappings but
 * does not overwrite or otherwise change this file.
 */
@Tag("todo-list")
@JsModule("./src/todo-list.js")
public class TodoList extends PolymerTemplate<TodoList.TodoListModel> {

    @Id("vaadinVerticalLayout")
	private VerticalLayout vaadinVerticalLayout;
	@Id("label")
	private Label label;
	@Id("vaadinHorizontalLayout")
	private HorizontalLayout vaadinHorizontalLayout;
	@Id("label1")
	private Label label1;
	@Id("vaadinListBox")
	private Element vaadinListBox;
	@Id("label2")
	private Label label2;
	@Id("vaadinListBox1")
	private Element vaadinListBox1;
	@Id("vaadinListBox2")
	private Element vaadinListBox2;
	@Id("vaadinListBox3")
	private Element vaadinListBox3;
	@Id("vaadinItem")
	private Element vaadinItem;
	@Id("vaadinItem1")
	private Element vaadinItem1;
	@Id("vaadinItem2")
	private Element vaadinItem2;
	@Id("vaadinItem3")
	private Element vaadinItem3;
	@Id("hr")
	private Hr hr;
	@Id("vaadinItem4")
	private Element vaadinItem4;
	@Id("hr1")
	private Hr hr1;
	@Id("vaadinCheckbox")
	private Checkbox vaadinCheckbox;
	@Id("vaadinCheckbox1")
	private Checkbox vaadinCheckbox1;
	@Id("vaadinCheckbox2")
	private Checkbox vaadinCheckbox2;
	@Id("vaadinCheckbox3")
	private Checkbox vaadinCheckbox3;

	/**
     * Creates a new TodoList.
     */
    public TodoList() {
        // You can initialise any data required for the connected UI components here.
    }

    /**
     * This model binds properties between TodoList and todo-list
     */
    public interface TodoListModel extends TemplateModel {
        // Add setters and getters for template properties here.
    }
}
